### PR TITLE
Add python version to runtime info

### LINKED ISSUE #47: Add Python version
Should we collect Python version to decide when to stop supporting the old versions?

### PR DESCRIPTION
**Pull request by mike0sv**
closes #47

### CODE REVIEW

File: src/iterative_telemetry/__init__.py [hunk 1]
```diff
@@ -146,8 +146,11 @@ def _runtime_info(self):
         """
         Gather information from the environment where DVC runs to fill a report
         """
+        major, minor, patch, *_ = sys.version_info
 
         return {
+            "python_version": f"{major}.{minor}.{patch}",
+            "python_version_minor": f"{major}.{minor}",
```
Comment (efiop) [inline src/iterative_telemetry/__init__.py:153]: What's the point of having both? The naming is confusing too.

Comment (efiop) [reply src/iterative_telemetry/__init__.py:153]: We might want to know what python implementation this is in the future, maybe worth making it a dict with misc python info right away?

Comment (efiop) [reply src/iterative_telemetry/__init__.py:153]: @mike0sv BQ?

Isn't there a way to parse the version properly from `major.minor.patch`? Or maybe store major/minor/patch separately. Having the same thing twice looks really odd. `python_version_minor` sounds like just `minor` to me, not `major.minor`. This is just really odd looking.

Comment (mike0sv) [reply src/iterative_telemetry/__init__.py:153]: BQ = big query

Doesnt make sense to store major separately - at least until python 4 is released :)
We can store major+minor and patch separately - joining is easier than splitting

Comment (efiop) [reply src/iterative_telemetry/__init__.py:153]: Sorry for the delay.

Storing `major.minor` and `patch` separately seems equally ugly. Looking at, for example, homebrew analytics, seems like they store their versions normally without duplicated odd version slices.

Googling around, seems like grouping by split string in bq is easily doable, but maybe I'm missing some complications?

**Review by efiop — APPROVED** [review]

**Pull request merged**